graphql/resolve/add_mutation_test.yaml

```diff
@@ -2741,43 +2743,6 @@
     message: |-
       failed to rewrite mutation payload because duplicate XID found: S1
 
-- name: "Duplicate XIDs in single mutation for Interface"
```

Comment:
Removed this test as it was wrong. As xid does not have interface=true flag. It is perfectly fine to have Student and Teacher with same xid.